reminder_handler.py

```diff
@@ -8,7 +8,7 @@
 
 logger = logging.getLogger(__name__)
 
-PATTERN = re.compile('^remind(?:\s+me)?\s+(.+?)\s*(to|:|that|about)\s+(.+?)\s*$', flags=re.I)
+PATTERN = re.compile('^rem(?:ind|(?:em)?ber)(?:\s+me)?\s+(.+?)\s*(to|:|that|about)\s+(.+?)\s*$', flags=re.I)
```

Comment:
```suggestion
PATTERN = re.compile('^(?:remind|remember|rember)(?:\s+me)?\s+(.+?)\s*(to|:|that|about)\s+(.+?)\s*$', flags=re.I)
```

Reply:
but the crypticness is part of the joke D: